tests/Doctrine/Tests/Inflector/Rules/Dutch/DutchFunctionalTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Tests\Inflector\Rules\LanguageFunctionalTest;
+
+class DutchFunctionalTest extends LanguageFunctionalTest
+{
+    /**
+     * @return list<array{string, string}>
+     */
+    public function dataSampleWords(): array
+    {
+        return [
+            ['schip', 'schepen'],
```

Comment:
I think we need some extra tests here.

```suggestion
            ['schip', 'schepen'],
            ['idee', 'ideeën'],
```

Reply:
hi you 2,

did swap my incorrectness and did add note for possible rule.

tests running again now.

Reply:
Hello! 👋 

Please also add: `['meer', 'meren'],`

Reply:
added but outcommented for now.